### PR TITLE
Ping RC when component is "attached"

### PR DESCRIPTION
**Pull request by stulees**
- Ping RC and handle response and error

### CODE REVIEW

**Comment by stulees** [comment]
@donnapep @rodrigopavezi Please review, thanks

File: rise-data.html [hunk 1]
```diff
@@ -43,11 +55,40 @@
...
+      },
+
+      /**
+       * Fires when an error is received from the ping request.
+       */
+      _handlePingError: function(e, resp) {
+        this._isCacheRunning = false;
+        this._pingReceived = true;
+
+        console.log("[rise-data] Ping Error");
```
Comment (donnapep) [inline rise-data.html:81]: @stulees Can we get rid of this or do you need it for now?

Comment (stulees) [reply rise-data.html:81]: @donnapep I can get rid of that console log in next PR

**Comment by donnapep** [comment]
@stulees K. 👍

**Pull request merged**